option_alpha_webhook.py

```diff
@@ -87,17 +94,18 @@ def option_alpha_trigger():
         # Step 3: Determine if we should trigger the trade or no-trade webhook
         if is_trade_recommended(impact_analysis):
             # If GPT suggests stability, trigger trade URL
-            success = trigger_option_alpha(TRADE_URL)
+            success = trigger_option_alpha(trade_url)
```

Comment:
NOTE: I noticed your old hardcoded webhook URLs are still visible in your commit history.  I would **_HIGHLY_** recommend you either kill those web hooks and create new ones or squash those changes out of your commit history so folks can't copy/paste your webhooks for their own usage, potentially pausing your bots.